src/bugmon_tc/monitor/tasks.py

```diff
@@ -137,34 +153,70 @@ def env(self) -> Dict[str, str]:
         if self.trace_dest:
             env_object["TRACE_ARTIFACT"] = str(self.trace_dest)
 
+        if self.bug.platform.system == "Windows":
+            env_object["MSYSTEM"] = "MINGW64"
+
         return env_object
 
     @property
     def scopes(self) -> List[str]:
         """Scopes applied to the task"""
-        scopes = super().scopes
-        scopes.extend(
-            [
-                "docker-worker:capability:disableSeccomp",
-                f"queue:get-artifact:project/fuzzing/bugmon/{self.monitor_path}",
-            ]
-        )
+        scopes = [
+            f"queue:get-artifact:project/fuzzing/bugmon/{self.monitor_path}",
+            "queue:scheduler-id:-",
+        ]
+        if self.bug.platform.system == "Linux":
+            scopes.extend(
+                [
+                    "docker-worker:capability:device:hostSharedMemory",
+                    "docker-worker:capability:device:loopbackAudio",
+                    "docker-worker:capability:disableSeccomp",
+                    "docker-worker:capability:privileged",
+                ]
+            )
 
         return scopes
 
     @property
     def task(self) -> Dict[str, Any]:
-        task = super().task
-        task["payload"]["capabilities"]["privileged"] = True
-        task["payload"]["capabilities"]["disableSeccomp"] = True
-
-        return task
+        """Processor task definition"""
+        if self._task is None:
+            self._task = super().task
+            if self.bug.platform.system == "Windows":
+                self._task["payload"]["command"] = [
+                    "set HOME=%CD%",
+                    "set ARTIFACTS=%CD%",
+                    "set PATH="
+                    + ";".join(
+                        [
+                            r"%CD%\msys64\opt\python",
+                            r"%CD%\msys64\opt\python\Scripts",
+                            r"%CD%\msys64\MINGW64\bin",
+                            r"%CD%\msys64\usr\bin",
+                            "%PATH%",
+                        ]
+                    ),
+                    "bugmon-process",
+                ]
+                self._task["payload"]["image"] = "mozillasecurity/bugmon-win:latest"
```

Comment:
This isn't valid for generic-worker. The image must be in `mounts`.

Reply:
Fixed via [5f1eb58](https://github.com/MozillaSecurity/bugmon-tc/pull/15/commits/5f1eb586fbe51f7f7704e9a87c6857d96cb59db8).